### PR TITLE
[Localization] Add localization in party-ui-handler

### PR DESCRIPTION
**Pull request by Ohmry**
## What are the changes?

Add localization of party-ui-handler

## Why am I doing these changes?

Option's name is only english. I think it needs to translate.

## What did change?

- Added `partyUiHandler` resources in i18n
- Added `party-ui-handler.ts` in each localization folder
- Added `partyUiHandler` property in each localization folder
- Translated them to Korean
- Changed code to localize option's name if includes in `localizedOptions` variable else use Util.toReadableString as before

#### Korean
image

#### Other languages
image

## How to test the changes?

1. Checkout this PR and change language
2. When battle startup choose change pokemon
3. Choose a pokemon you want to send out.

## Checklist
- [X] There is no overlap with another PR?
- [X] The PR is self-contained and cannot be split into smaller PRs?
- [X] Have I provided a clear explanation of the changes?
- [X] Have I tested the changes (manually)?
    - [X] Are all unit tests still passing? (`npm run test`)
- [X] Are the changes visual?
  - [X] Have I provided screenshots/videos of the changes?

### CODE REVIEW

**Comment by Ohmry** [comment]
`localizedOptions` variable is temporay variable. I think we should change this code to localize all option names and then localize the names without the `Utils.toReadableString` method.

File: src/locales/de/party-ui-handler.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { SimpleTranslationEntries } from "#app/plugins/i18n";
+
+export const partyUiHandler: SimpleTranslationEntries = {
+  "SEND_OUT": "Send Out",
+  "SUMMARY": "Summary",
+  "CANCEL": "Cancel",
+  "RELEASE": "Release",
+  "APPLY": "Apply",
+  "TEACH": "Teach"
```
Comment (CodeTappert) [inline src/locales/de/party-ui-handler.ts:9]:
```suggestion
  "SEND_OUT": "Einwechseln",
  "SUMMARY": "Bericht",
  "CANCEL": "Abbrechen",
  "RELEASE": "Freilassen",
  "APPLY": "Anwenden",
  "TEACH": "Erlernen"
```

Comment (Ohmry) [reply src/locales/de/party-ui-handler.ts]: Done (dba647c)

**Comment by sodaMelon** [comment]
hi. i will check this pull request. wait a minute...

File: src/locales/ko/party-ui-handler.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { SimpleTranslationEntries } from "#app/plugins/i18n";
+
+export const partyUiHandler: SimpleTranslationEntries = {
+  "SEND_OUT": "교체한다",
+  "SUMMARY": "능력치를 본다",
+  "CANCEL": "그만둔다",
+  "RELEASE": "놓아준다",
+  "APPLY": "사용한다",
+  "TEACH": "가르친다"
+} as const;
```
Comment (sodaMelon) [inline src/locales/ko/party-ui-handler.ts:10]: 확인했습니다. 한국어 의미에 부합하도록 정확하게 번역하셨네요.

**Review by sodaMelon — APPROVED** [review]
LGTM 👍

Now Korean users can play games more comfortably.
thank you!!

**Review by Tempo-anon — APPROVED** [review]

**Comment by josericardo-fo** [comment]
Is there a localization for the "Unsplice" option?

**Comment by Ohmry** [comment]
> Is there a localization for the "Unsplice" option?

Not yet, you can created new PR about that if you want.

**Pull request merged**